frontend/js/src/artist/ArtistPage.tsx

```diff
@@ -125,12 +125,15 @@ export default function ArtistPage(): JSX.Element {
   const [expandPopularTracks, setExpandPopularTracks] = React.useState<boolean>(
     false
   );
+  const [expandDiscography, setExpandDiscography] = React.useState<boolean>(
```

Comment:
I believe we should reset the states when the user redirects from one artist page to another. This issue is also there in the state `expandPopularTracks`.

Right now, when you navigate from one page to other, the expandDiscography state retain their present values.